examples/variational_autoencoder.py

```diff
@@ -1,111 +1,205 @@
-'''This script demonstrates how to build a variational autoencoder with Keras.
+'''Example of VAE on MNIST dataset using MLP
 
- #Reference
+This VAE has a modular design. The encoder, decoder and vae
+are 3 models that share weights. After training vae,
+the encoder can be used to  generate latent vectors.
+The decoder can be used to generate MNIST digits by sampling the
+latent vector from a gaussian dist with mean=0 and std=1.
 
- - Auto-Encoding Variational Bayes
-   https://arxiv.org/abs/1312.6114
+[1] Kingma, Diederik P., and Max Welling.
```

Comment:
Prefer the previous `# References` formatting

Reply:
fixed but I retained the authors